recipes/compilers/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: compilers-metapackages
+  version: 1.0.0
+
+build:
+  number: 0
+
+outputs:
+  - name: c-compiler
+    requirements:
+      run:
+        - {{ compiler('c') }}
```

Comment:
I thought this would add soft-links? Or will we do that in the feedstock?

Reply:
I wasn't sure if we had agreed on that, and any am not really sure how to do that...

Reply:
Also, this might be for @conda-forge/core, but should this be `{{ pin_compatible(compiler('c'), exact=True) }}`?

Reply:
> I wasn't sure if we had agreed on that, and any am not really sure how to do that..

OK no worries, someone else who wants them can add them

Reply:
I can give it a go and see what I come up with.